### PR TITLE
Fix multiple events subscription

### PR DESCRIPTION
**Pull request by tadatuta**
This PR also addresses #1526 for bem-core version without jQuery.

### CODE REVIEW

File: common.blocks/i-bem-dom/__events/i-bem-dom__events.js [hunk 1]
```diff
@@ -52,38 +52,44 @@ var undef,
...
+                        });
+                    }
+                }.bind(this),
+                handler;
 
-                bindDomNodes.forEach(function(domNode) {
-                    domNode.addEventListener(event, handler, false);
-                });
-            }
+            typeof e === 'string'?
```
Comment (veged) [inline common.blocks/i-bem-dom/__events/i-bem-dom__events.js]: it's better to do this in very first lines of this function — just check if `e` is a string and contains spaces and do `forEach` of `this.on` (we do similar https://github.com/bem/bem-core/blob/v4/common.blocks/i-bem-dom/__events/i-bem-dom__events.js#L226)

File: common.blocks/i-bem-dom/__events/i-bem-dom__events.js [hunk 2]
```diff
@@ -115,26 +121,32 @@ var undef,
...
-                        domNode.removeEventListener(event, handler);
-                    });
-                }
+                            bindDomNodes.forEach(function(domNode) {
+                                domNode.removeEventListener(event, handler);
+                            });
+                        }
+                    }.bind(this);
+
+                typeof e === 'string'?
```
Comment (veged) [inline common.blocks/i-bem-dom/__events/i-bem-dom__events.js]: same stuff like with `on` — we can make smaller diff

**Comment by tadatuta** [comment]
@veged ⬆️

File: common.blocks/i-bem-dom/__events/i-bem-dom__events.js [hunk 3]
```diff
@@ -52,6 +52,14 @@ var undef,
          * @returns {EventManager} this
          */
         on : function(e, data, fn, _fnCtx, _isOnce) {
+            if(typeof e === 'string' && e.indexOf(' ') > -1) {
+                e.split(' ').forEach(function(evnt) {
```
Comment (veged) [inline common.blocks/i-bem-dom/__events/i-bem-dom__events.js]: `evEnt`

**Review by veged — APPROVED** [review]

**Pull request merged**